### ISSUE TITLE
F17 being removed has left gaps in WCAG rules

### ISSUE THREAD

**Issue by jribbens** [w3c/wcag#1491]
F17 was removed because F77 overlapped with it. However F17 included 4 tests, and only 1 of these overlapped with F77, so its removal lost 3 important tests that still need to be covered by WCAG (non-unique accesskeys, and various types of references to IDs where the ID does not exist). F17 should be reinstated (perhaps with its test 1 removed).

**Comment by patrickhlauke** [comment]
duplicate of https://github.com/w3c/wcag/issues/1492

**Comment by jribbens** [comment]
Apologies, no idea how this duplicate happened - github responded with a 404 error page the first time I opened it (!)

**Comment by patrickhlauke** [comment]
hey, no biggie. sometimes our tools fail us :)
